doc/manuals/admin/logs.md

```diff
@@ -16,6 +16,14 @@ When starting the Orion context broker, if a previous log file exist:
 -   If **-logAppend** is not used, then the existing file is renamed,
     appending the text ".old" to its name.
 
+The `-logLevel` option allows to chose which error messages will be printed in the log:
```

Comment:
Strictly speaking, `-logLevel` is not part of this PR, but I have added this in order to advance a little bit with documentation.